src/index.html

```diff
@@ -1,13 +1,83 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__logo-container">
+        <img
+          src="./images/imac.jpeg"
+          alt="MacAPPLE A1419 iMac 27"
+          class="card__logo"
+        >
+      </div>
+
+      <div class="card__content-container">
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+        <div class="card__rating-container">
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__image">
+              </div>
+
+              <div class="stars__image">
+              </div>
+
+              <div class="stars__image">
+              </div>
+
+              <div class="stars__image">
+              </div>
+
+              <div class="stars__image">
+              </div>
+            </div>
+
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+        </div>
```

Comment:
Consider moving the content of card__reviews inside card__rating-container to avoid extra div.